dask/array/core.py

```diff
@@ -2660,6 +2660,10 @@ def bincount(x, weights=None, minlength=None):
 
     return Array(dsk, name, chunks, dtype)
 
+@wraps(np.digitize)
+def digitize(a, bins, right=False):
+    func = partial(np.digitize, bins=bins, right=right)
+    return elemwise(func, a)
```

Comment:
Yeah, I have no intention of abandoning the array interface.  It is, as you say, useful in many contexts.

I was merely suggesting setting `map_partitions = map_blocks` in order to avoid this naming confusion in the future.